app/assets/src/components/SelectSamples.jsx

```diff
@@ -0,0 +1,1265 @@
+import React from 'react';
+import axios from 'axios';
+import ReactDOM from 'react-dom';
+import moment from 'moment';
+import $ from 'jquery';
+import Tipsy from 'react-tipsy';
+import Materialize from 'materialize-css';
+import SortHelper from './SortHelper';
+import numberWithCommas from '../helpers/strings';
+import ProjectSelection from './ProjectSelection';
```

Comment:
Do you have to copy the jsx over? Can you use the sample component but with the flag turned off/on on select/home page? So you can keep working on the improvements and when the time comes, we just switch the flag on without trying to merge the divergent versions.